presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/HadoopKerberos.java

```diff
@@ -69,4 +76,10 @@ public void extendEnvironment(Environment.Builder builder)
             container.withCopyFileToContainer(forHostPath(configDir.getPath("tempto-configuration.yaml")), CONTAINER_TEMPTO_PROFILE_CONFIG);
         });
     }
+
+    @Override
+    public List<EnvironmentExtender> getDependencies()
```

Comment:
I am not convinced we need this class at all. 
Why not just call `hadoop.extend()` in `extendEnvironment`.

(possibly it was discussed alrady)...

Reply:
I don't understand.

> I am not convinced we need this class at all.

Do you mean we don't need `HadoopKerberos`?

Can you please elaborate?

Reply:
I meant method. sorry.

Reply:
> Why not just call hadoop.extend() in extendEnvironment.

That was the first approach. Notice that there might be other `extendEnvironment` that could also depend on `hadoop`, in such case `haddop` would be called multiple times.

Reply:
> > Why not just call hadoop.extend() in extendEnvironment.
> 
> That was the first approach. Notice that there might be other `extendEnvironment` that could also depend on `hadoop`, in such case `haddop` would be called multiple times.

Yeah - I was afraid that is the case.